keras/kokoro/github/ubuntu/gpu/build.sh

```diff
@@ -46,10 +41,39 @@ fi
 if [ "$KERAS_BACKEND" == "jax" ]
 then
    echo "JAX backend detected."
+   pip install -r requirements-jax-cuda.txt --progress-bar off
+   pip uninstall -y keras keras-nightly
+   python3 -c 'import jax;print(jax.__version__);print(jax.default_backend())'
+   # Raise error if GPU is not detected.
+   python3 -c 'import jax;assert jax.default_backend().lower() == "gpu"'
+
+   # TODO: keras/layers/merging/merging_test.py::MergingLayersTest::test_sparse_dot_2d Fatal Python error: Aborted
+   # TODO: FAILED keras/layers/preprocessing/feature_space_test.py::FeatureSpaceTest::test_saving
+   # TODO: keras/trainers/data_adapters/py_dataset_adapter_test.py::PyDatasetAdapterTest::test_basic_flow0 Fatal Python error: Aborted
+   # TODO: FAILED keras/distribution/distribution_lib_test.py
+   pytest keras --ignore keras/applications \
+               --ignore keras/layers/merging/merging_test.py \
+               --ignore keras/layers/preprocessing/feature_space_test.py \
+               --ignore keras/trainers/data_adapters/py_dataset_adapter_test.py \
+               --ignore keras/distribution/distribution_lib_test.py \
+               --cov=keras
 fi
 
 # TODO: Add test for PyTorch
 if [ "$KERAS_BACKEND" == "torch" ]
 then
    echo "PyTorch backend detected."
+   pip install -r requirements-torch-cuda.txt --progress-bar off
+   pip uninstall -y keras keras-nightly
+   python3 -c 'import torch;print(torch.__version__);print(torch.cuda.is_available())'
+   # Raise error if GPU is not detected.
+   python3 -c 'import torch;assert torch.cuda.is_available()'
+
+   # TODO: Fix the failing Torch GPU CI tests.
+   # TODO: nn_test failures are on correctness tests.
+   pytest keras --ignore keras/applications \
+               --ignore keras/layers/preprocessing/feature_space_test.py \
+               --ignore keras/layers/reshaping/flatten_test.py \
+               --ignore keras/ops/nn_test.py \
```

Comment:
This one is quite large! Among the different failures, we should prioritize this one.